src/main/java/org/jboss/set/ValidationCommand.java

```diff
@@ -0,0 +1,147 @@
+                    }
+                }
+            }
+        } catch (IOException | URISyntaxException e) {
+            errorAndPrint(e, "Unable to open anonymous connection to GitHub or parse URL of the repository");
+            throw new RuntimeException(e);
+        }
+    }
+
+    private boolean validateConfig(LotteryConfig lotteryConfig) {
+        Map<String, Collection<String>> projectComponentsMap = new HashMap<>();
+        Set<String> participants = new HashSet<>();
+        boolean valid = true;
+
+        for (LotteryConfig.Participant participant : lotteryConfig.participants()) {
+
+            if (participant.maxIssues() != null && participant.maxIssues() <= 0) {
+                errorAndPrint("Participant %s has defined non-positive number of max issues".formatted(participant.email()));
+                valid = false;
+            }
+
+            for (LotteryConfig.Participant.Project project : participant.projects()) {
+                if (participants.contains(participant.email())) {
+                    errorAndPrint("Two identical participants detected %s".formatted(participant.email()));
+                }
+                participants.add(participant.email());
+
+                if (project.participation().maxIssues() <= 0) {
+                    errorAndPrint("Participant %s has defined non-positive number of max issues for project %s"
+                            .formatted(participant.email(), project.project()));
+                    valid = false;
+                }
+
+                // we assume a valid project has some components
+                Collection<String> components = projectComponentsMap.computeIfAbsent(project.project(), projectKey -> {
+                    Project jiraProject = getJiraEndpoint().getClient().getProjectClient().getProject(projectKey).claim();
+                    return StreamSupport.stream(jiraProject.getComponents().spliterator(), false).map(BasicComponent::getName)
+                            .map(String::toLowerCase).toList();
+                });
+
+                if (components.isEmpty()) {
+                    errorAndPrint(
+                            "For participant %s and project %s no components were found on JIRA. A typo in project is thus expected."
+                                    .formatted(participant.email(), project.project()));
+                    valid = false;
+                }
+
+                if (project.components() == null) {
+                    continue;
+                }
+
+                Collection<String> lowerCasedComponents = project.components().stream().map(String::toLowerCase).toList();
+                if (!components.containsAll(lowerCasedComponents)) {
+                    errorAndPrint(
+                            "For participant %s and project %s some components were found under the project based on JIRA. A typo in one or more components is thus expected."
```

Comment:
What? You expect some typo?

Reply:
Yes, if we know a project exists, however, we are unable to find all the components a user specified in the config file. Thus, we expect a typo in the user defined components.

Reply:
Just rewrite the sentence to "You probably made a typo" because this means that you are expecting me to make some typo yet.

Reply:
@The-Huginn this still doesn't make sense, 
you probably made a typo"